.claude/review-process.md

```
@@ -1,8 +1,18 @@
 # PR 리뷰 프로세스
 
-## 리뷰 위임 방법
+## 전체 리뷰 흐름 (순서)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)